src/pre.ts

```diff
@@ -22,3 +28,14 @@ const onSensorChange = () =>
   );
 window.board = createBoard(onSensorChange);
 window.constants = constants;
+window.conversions = {
+  convertAudioBuffer: (source: number, target: AudioBuffer) => {
+    const channel = target.getChannelData(0);
+    const heap = window.HEAPU8;
+    for (let i = 0; i < channel.length; ++i) {
+      // Convert from uint8 to -1..+1 float.
+      channel[i] = (heap[source + i] / 255) * 2 - 1;
+    }
+    return target;
+  },
```

Comment:
We can put this in its own file and use typeof to get the types right without restating them above.